src/main/c/ssl.c

```diff
@@ -1618,6 +1753,7 @@ TCN_IMPLEMENT_CALL(jlong, SSL, getTime)(TCN_STDARGS, jlong ssl)
     if (ssl_->session == NULL) {
         // BoringSSL does not protect against a NULL session. OpenSSL
         // returns 0 if the session is NULL, so do that here.
+        // TODO(nmittler): should we throw here?
```

Comment:
@normanmaurer Just to be clear, is your concern of using tcnative with BoringSSL and an old version of netty? With the new version of Netty, throwing would be the right thing to do, since it would expose a programming error.